docs/cudf/source/conf.py

```diff
@@ -202,14 +208,51 @@
 autoclass_content = "init"
 
 # Config AutoStructify
-github_doc_root = 'https://github.com/rtfd/recommonmark/tree/master/doc/'
+github_doc_root = "https://github.com/rtfd/recommonmark/tree/master/doc/"
+
+# Replace API shorthands with fullname
+reftarget_aliases = {
+    "cudf.Series": ("cudf.core.series.Series", "cudf.Series"),
+    "cudf.Index": ("cudf.core.index.Index", "cudf.Index"),
+    "cupy.core.core.ndarray": ("cupy.ndarray", "cupy.ndarray"),
```

Comment:
Both elements of the tuple here are the same -- is that intended?

Reply:
Yeah, because on `cupy` sphinx doc they remapped `cupy.core.core.ndarray` to `cupy.ndarray`. This is to link to the right entry. The second item is just the display text.